src/datetimepicker.android.js

```diff
@@ -109,7 +113,14 @@ export default function RNDateTimePicker(props: AndroidNativeProps) {
               break;
 
             case TIME_SET_ACTION:
-              event.nativeEvent.timestamp = date.setHours(hour, minute);
+              date.setHours(hour, minute);
+              if (timeZoneOffsetInMinutes !== undefined) {
+                const offset =
+                  date.getTimezoneOffset() * 60000 +
```

Comment:
could you please extract `60000` and give it a name? I'm aiming at making the code readability as good as possible for newcomers and it this would help understanding it, thanks